config/development.json

```diff
@@ -43,6 +43,7 @@
 		"comments/moderation-tools-in-posts": true,
 		"comments/management": true,
 		"comments/management/comment-view": true,
+		"comments/management/m3-design": true,
```

Comment:
Maybe keep this off until we add in the actions bar? We can also test things in a non-hacky way with `ENABLE_FEATURES=comments/management/m3-design npm start`